src/Entity/CitizenProjectCategorySkill.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\CitizenProjectCategorySkillRepository")
+ * @ORM\Table(
+ *   name="citizen_project_category_skills",
+ * )
+ *
+ * @UniqueEntity(
+ *     fields={"category", "skill"},
+ *     errorPath="skill",
+ *     message="This skill is already in use on that category."
+ * )
+ */
+class CitizenProjectCategorySkill
+{
+    /**
+     * @var int|null
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     */
+    private $id;
+
+    /**
+     * @var CitizenProjectCategory
+     *
+     * @ORM\ManyToOne(targetEntity="CitizenProjectCategory", inversedBy="categorySkills")
```

Comment:
General comment, do we really need all these bidirectional relationships ? Isn't it better to stay with unidirectional as much as possible ?

Reply:
agree. this one needs to be bidirectional though.